README.md

```diff
@@ -1,3 +1,105 @@
-# imgix-api-js
+<!-- ix-docs-ignore -->
 
-A JS library that wraps the imgix public API
+![imgix logo](https://assets.imgix.net/sdk-imgix-logo.svg)
+
+`imgix-management-js` is a JavaScript client library for making requests against the [imgix](https://www.imgix.com/) management API.
+
+[![Build Status](https://travis-ci.org/imgix/imgix-management-js.svg?branch=master)](https://travis-ci.org/imgix/imgix-management-js)
+
+---
+
+<!-- /ix-docs-ignore -->
+
+## Installation
+
+`npm install imgix-management-js`
+
+`imgix-management-js` exposes a class `ImgixAPI`, which can be loaded in the following way:
+
+### CommonJS
+
+```js
+const ImgixAPI = require("imgix-management-js");
+
+const imgix = new ImgixAPI({
+    apiKey: `${yourApiKey}`
+});
+```
+
+## Configuration
+
+The following options can be used when creating an instance of `ImgixAPI`:
+
+- `apiKey`: String, required. The token used to authenticate API requests.
+- `version`: Integer. The version of the API that will be requested against. Defaults to `1`.
+
+## API
+
+### `ImgixAPI.request(path, options)`
+
+- `path`: String, required. A partial path representing the relative URL endpoint to request. This will typically require a `sourceId`.
+- `options`: A combination of User and [HTTP(S)](#Options) options.
```

Comment:
We definitely need to be more specific about exactly what options are available here.